media-sound/teamspeak-server/teamspeak-server-3.3.0.ebuild

```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit systemd user
+
+DESCRIPTION="A server software for hosting quality voice communication via the internet"
+HOMEPAGE="https://www.teamspeak.com/"
+SRC_URI="amd64? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/teamspeak3-server_linux_amd64-${PV}.tar.bz2 )
+	x86? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/teamspeak3-server_linux_x86-${PV}.tar.bz2 )"
+
+LICENSE="LGPL-2.1 teamspeak3"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE="doc mysql tsdns"
+
+RESTRICT="mirror strip"
+
+S="${WORKDIR}/teamspeak3-server_linux"
+
+QA_PREBUILT="opt/teamspeak3-server/libmariadb.so.2
+	opt/teamspeak3-server/libts3db_mariadb.so
+	opt/teamspeak3-server/libts3db_sqlite3.so
+	opt/teamspeak3-server/libts3_ssh.so
+	opt/teamspeak3-server/ts3server
+	opt/teamspeak3-server/tsdnsserver"
+
+pkg_setup() {
+	enewgroup teamspeak
+	enewuser teamspeak -1 -1 /opt/teamspeak3-server teamspeak
+}
+
+src_unpack() {
+	unpack ${A}
+
+	mv teamspeak3-server_linux_$(usex amd64 amd64 x86) teamspeak3-server_linux || die
```

Comment:
If you need to rename it anyway, why not use a name that would render redefining S unnecessary?

Reply:
Done. Good idea!